extension/src/contentScript/helpers/setIsConnected.ts

```diff
@@ -1,7 +1,6 @@
-export const embedLyraApi = () => {
-  const url = chrome.extension.getURL("lyraApi.min.js");
+export const setIsConnected = () => {
   const script = document.createElement("script");
-  script.src = url;
+  script.textContent = "window.lyra = true;";
```

Comment:
instead of putting the whole API on window, creating a flag just to let the client app know that Lyra is available

Reply:
They will check this flag by calling lyraApi.isConnected()